openid-sharedsignals-framework-1_0.md

```diff
@@ -2218,9 +2220,9 @@ endpoint_url
 
 authorization_header
 
-> The HTTP Authorization header that the Transmitter MUST set with each event
-  delivery, if the configuration is present. The value is optional and it is set
-  by the Receiver.
+> If the endpoint_url requires authorization, the receiver SHOULD provide
```

Comment:
Could you please remove this from the PR, I'm addressing this in the "refactor section 10 PR"